src/acvp_lcl.h

```diff
@@ -65,6 +65,17 @@
 #define ACVP_ALG_HASHDRBG            "hashDRBG"
 #define ACVP_ALG_HMACDRBG            "hmacDRBG"
 #define ACVP_ALG_CTRDRBG             "ctrDRBG"
+#define ACVP_ALG_HMAC_SHA1           "HMAC-SHA1"
+#define ACVP_ALG_HMAC_SHA2_224       "HMAC-SHA2-224"
+#define ACVP_ALG_HMAC_SHA2_256       "HMAC-SHA2-256"
+#define ACVP_ALG_HMAC_SHA2_384       "HMAC-SHA2-384"
+#define ACVP_ALG_HMAC_SHA2_512       "HMAC-SHA2-512"
+#define ACVP_ALG_HMAC_SHA2_512_224   "HMAC-SHA2-512/224"
```

Comment:
Do we need 512/224 and 512/256 in the tables and case statements like SHA3 ?
And like SHA3 we return unsupported on registration until someone(not us) implements it ?